Packages/Sources/RootView/Home.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  SwiftUIView.swift
+//
+//
+//  Created by HH on 09/01/2024.
+//
+
+import NodeCryptoCore
+import SwiftUI
+
+struct Home: View {
+  @State private var activeTab: Tab = .search
+  @Namespace var animation
+
+  var body: some View {
+    VStack(spacing: 0) {
+      TabView(selection: $activeTab) {
+        Color.red
+          .tag(Tab.home)
+          .toolbar(.hidden, for: .tabBar)
+
+        Text("Services")
+          .tag(Tab.search)
+          .toolbar(.hidden, for: .tabBar)
+
+        Text("Notifications")
+          .tag(Tab.notifications)
+          .toolbar(.hidden, for: .tabBar)
+
+        Text("Profile")
+          .tag(Tab.profile)
+          .toolbar(.hidden, for: .tabBar)
+      }
+
+      CustomTabBar()
+    }
+  }
+
+  @ViewBuilder
+  func CustomTabBar(tint: Color = .blue, inactiveTint: Color = .blue) -> some View {
+    HStack(spacing: 0) {
+
+      TabItem(
```

Comment:
This code is repeated, please extract a create a function e.g. like this `makeTabItem(.home)`. Please note "make" you can read about it in aforementioned Swift API design guidelines.